tensorflow/python/ops/math_ops.py

```diff
@@ -101,6 +101,112 @@
 linspace = gen_math_ops.lin_space
 nextafter = gen_math_ops.next_after
 
+
+@tf_export("linspace_nd")
```

Comment:
Do we need a new API entry point or can we just add axis to tf.linspace?

Reply:
I tried to do that locally, but it collides with the automatically exported `linspace` kernel at `tensorflow/core/kernels/sequence_ops.cc`, deleting it did not work, I got an error which was still pointing to the `LinSpaceOp` kernel somehow - do you know how can I remove the export of the `LinSpaceOp` kernel?

Reply:
@hristo-vrigazov you can add `visibility: HIDDEN` to
```
tensorflow/core/api_def/base_api/api_def_LinSpace.pbtxt 
```

That works in the past (haven't touched this area for some time so I don't know if this still holds).

Reply:
@yongtang Thanks! It is exactly what was needed. :) @alextp I exposed the new function under linspace, but `axis` is the last parameter (as we spoke in the previous PR about backward compatibility. Let me know if this is ok and I can resolve this point :)

Reply:
Yes this is ok, resolving!